### ISSUE TITLE
Accessibility: tooltip text is not selectable

### ISSUE THREAD

**Issue by rokalm** [ng-bootstrap/ng-bootstrap#4168]
### Bug description

Tooltip content is not selectable, when use tries to hover on tooltip itself
1. Hover over on tooltip https://ng-bootstrap.github.io/#/components/tooltip/examples
2. Try to select text

Expected: see behavior here https://bootstrap-vue.org/docs/components/tooltip

WSAG Requirement Level AA https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html:
```
Hoverable
If pointer hover can trigger the additional content, then the pointer can be moved over the additional content without the additional content disappearing;
```

**Comment by maxokorokov** [comment]
I think you're asking to add the ability to make tooltips and popovers interactive/hoverable. Something along those lines:

```html
<button ngbTooltip="hi" [hoverable]="true" />
```

Similar to the vue-bootstrap's "interactive" tooltips → https://bootstrap-vue.org/docs/components/tooltip

I think it's a valid request and should actually be the default behaviour.
You could "simulate" with something like this → https://stackblitz.com/edit/angular-1dsbf1?file=src%2Fapp%2Ftooltip-triggers.html

But we should add this option.

**Comment by senthilchandran** [comment]
@maxokorokov Any update on this request? Is there an ETA on when this feature will be available to consume?

Thanks!

**Comment by daiscog** [comment]
@maxokorokov this is actually a serious concern for accessibility.  It's not just about being able to "select" the text of the tooltip. Some partially-sighted users may use screen magnifiers to enlarge areas under the cursor; these users may need to move the cursor away from the trigger element and over the tooltip itself in order to bring the tooltip under the magnified area so they can read it.  If the tooltip disappears immediately on leaving the trigger element, then it is complete inaccessible to these users, which will lead to a very frustrating user experience.

So I completely agree with your opinion that it "should actually be the default behaviour".  This project should strive to be accessible by default, and not leave it up to consumers to try to hack accessibility on top through manual workarounds.  IMO an issue seeking to address a lack of accessibility OOTB should be considered a **bug** not a feature request.

Could the team please consider prioritising a resolution for this issue soon?
